### PR TITLE
Add StarkNet + Ethereum clients

### PR DESCRIPTION
**Pull request by akegaviar**
Added StarkNet docs.
Added Ethereum Geth + Erigon.

### CODE REVIEW

**Review by NikitosnikN — APPROVED** [review]

File: docs/operations/bsc/clients.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  - name: keywords
+    content: bsc binance node geth erigon difference
+---
+
+# Clients
+
+Chainstack supports deploying a dedicated Binance Smart Chain archive node on the mainnet in the following client implementations:
+
+* Geth — the [Go Ethereum](https://github.com/bnb-chain/bsc) implementation.
+* Erigon — the [Erigon](https://github.com/bnb-chain/bsc-erigon) implementation.
```
Comment (easeev) [inline docs/operations/bsc/clients.md:14]: BSC is merged into main repo https://github.com/ledgerwatch/erigon

File: docs/operations/bsc/clients.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    content: Learn the difference between a Binance Smart Chain Geth node and an Erigon node.
+  - name: keywords
+    content: bsc binance node geth erigon difference
+---
+
+# Clients
+
+Chainstack supports deploying a dedicated Binance Smart Chain archive node on the mainnet in the following client implementations:
+
+* Geth — the [Go Ethereum](https://github.com/bnb-chain/bsc) implementation.
```
[the remark this answers is not in the document]
Comment (easeev) [reply docs/operations/bsc/clients.md:13]: BTW this article applies to Ethereum, that's where we have both Geth and Erigon as options

[the remark this answers is not in the document]
Comment (akegaviar) [reply docs/operations/bsc/clients.md]: Changed

File: docs/operations/bsc/clients.md (hunk 1 above)
Comment (akegaviar) [reply docs/operations/bsc/clients.md]: Changed

File: docs/platform/join-a-public-network.md [hunk 3]
```diff
@@ -47,6 +47,7 @@ The node status will change from **Pending** to **Running** once deployed.
 1. Under **Blockchain network**, select **Mainnet** or **Testnet**. Click **Next**.
 1. Under **Type**, select whether to run a [shared](/glossary/shared-node) or a [dedicated](/glossary/dedicated-node) node.
 1. Under **Mode**, select whether to run a full node or an archive node. See [Modes](/operations/bsc/modes).
+1. If you are deploying a dedicated archive node, under **Client**, select whether to run a Geth node or an Erigon node. See [Clients](/operations/bsc/clients).
```
[the remark this answers is not in the document]
Comment (akegaviar) [reply docs/platform/join-a-public-network.md]: Changed

**Comment by akegaviar** [comment]
> There should be choice between Geth and Erigon for Ethereum archive

@NikitosnikN there's no choice in the selection wizard.

**Comment by easeev** [comment]
> @NikitosnikN there's no choice in the selection wizard.

So what's the status here @PekopT?

**Comment by PekopT** [comment]
@easeev selector for Erigon in Ethereum archive will be added next week EOW

**Pull request merged**